packages/agoric-cli/lib/main.js

```diff
@@ -218,6 +221,22 @@ const main = async (progname, rawArgs, powers) => {
       return subMain(startMain, ['start', profile, ...args], opts);
     });
 
+  program
+    .command('update')
+    .description('Update the Agoric SDK.')
```

Comment:
Thanks for this addition to the agoric CLI.  I appreciate the code, but to make it general enough to merge will take some more work.

The CLI code has been factored in an ocap style.  Try having a look at `lib/entrypoint.js` to see the `powers` third argument to `main`.  Those system APIs are passed to `main` from `entrypoint.js` instead of importing things that implicitly grant authority.